### ISSUE TITLE
Django 1.10 - rqworker: error: unrecognized arguments

### ISSUE THREAD

**Issue by GhalebKhaled** [ui/django-rq#195]
When I upgraded to `django 1.10`,` django-rq 0.9.2` seems to be not compatible with `django 1.10` changes and it raises the following error:
```
usage: manage.py rqworker [-h] [--version] [-v {0,1,2,3}]
                          [--settings SETTINGS] [--pythonpath PYTHONPATH]
                          [--traceback] [--no-color]
                          [--worker-class WORKER_CLASS] [--pid PID]
                          [--burst BURST] [--name NAME]
                          [--queue-class QUEUE_CLASS]
                          [--worker-ttl WORKER_TTL]
manage.py rqworker: error: unrecognized arguments: automation
```

According to change log for 0.9.2 release compatibility issue should be fixed, but I'm still facing this issue. When I install `django-rq` from master branch, it works fine.

My problem that one of the dependancies library requires `django-rq` >= `0.9.1` which leads to install `django-rq 0.9.2` from pypi. Even if I put the git repo in requirements.txt, the pip see the git version as same as the pypi release of 0.9.2 which makes it just ignore the git version (as it assume its already installed)

Can you please increment current version on git master branch to 0.9.3? any maybe create new pypi release?

**Comment by selwin** [comment]
Will tag a release this weekend.

**Comment by frisellcpl** [comment]
Did this issue get solved/any release tagged?

**Comment by nealtodd** [comment]
@frisellcpl solved in #180 but awaiting a 0.9.3 release

**Comment by selwin** [comment]
Just released 0.9.3 on PyPI. Sorry for the wait.

**Comment by nealtodd** [comment]
Thanks @selwin, much appreciated. (Perfect timing for me as I only came across the issue yesterday :)
